react-common/components/share/ShareInfo.tsx

```diff
@@ -311,6 +352,10 @@ export const ShareInfo = (props: ShareInfoProps) => {
                             </div>
                             <div className="project-share-actions">
                                 <div className="project-share-social">
+                                <Button className="square-button gray mobile-portrait-hidden"
```

Comment:
This needs to go behind a pxtarget flag, something like `pxt.appTarget.appTheme.shareToKiosk` - we won't be wanting this in micro:bit, for example~